package-system/assimp/build_assimp_mac.sh

```diff
@@ -0,0 +1,26 @@
+#
+# Copyright (c) Contributors to the Open 3D Engine Project.
+# For complete copyright and license terms please see the LICENSE at the root of this distribution.
+# 
+# SPDX-License-Identifier: Apache-2.0 OR MIT
+#
+#
+
+cmake -S temp/src -B temp/build \
+    -DCMAKE_CXX_COMPILER=clang++ \
```

Comment:
Is mac using clang and linux using gcc?

Reply:
I'm not too sure, I'm not very familiar with either platform, these two scripts are what I was able to get working after a lot of trial and error and looking at all the other 3rd party libraries.

Reply:
Reason I ask is because it seems you should be able to build with the same args for both mac and linux, so you can just use one script for both platforms. I just recently did the same thing for SQLite (it wasn't cmake though).

Reply:
After I figure out this Windows failure that's occurring with the patch, I'll try the Linux script on mac then and see what I get. When I tried to use this command on Linux it couldn't find Clang, I ended up spending a lot of time on the internal wiki looking up things to try until some combination of "apt get" and that iteration of the script worked.